libindy/src/api/mod.rs

```diff
@@ -185,6 +185,9 @@ pub enum ErrorCode
     // Call pool.indy_set_protocol_version to set correct Protocol version.
     PoolIncompatibleProtocolVersion = 308,
 
+    // Item not found on ledger.
```

Comment:
@dkulic please add new error code to `libindy/includes/`

Reply:
Will do.